virtme/guest/virtme-init

```diff
@@ -364,14 +366,14 @@ fi
 if [[ -n "${virtme_console}" ]]; then
     consdev=${virtme_console}
 else
-    consdev="`grep ' ... (.C' /proc/consoles  |cut -d' ' -f1`"
+    consdev="$(grep ' ... (.C' /proc/consoles  |cut -d' ' -f1)"
```

Comment:
(while at it, could you fix the spaces around `|` please?)

Reply:
There were 2 spaces before the pipe, I guess it was an accident.

Reply:
Ah right, thought you would mean ` |cut` vs. ` | cut`